src/components/molecules/DateTimePicker/DateTimePicker.tsx

```diff
@@ -32,9 +33,12 @@ const DateTimePicker = forwardRef<HTMLInputElement, DateTimePickerProps>(
       className,
       minDate,
       maxDate,
+      onDateTimeChange,
     } = props
 
     const { t } = useTranslation()
+    const [isModalOpen, setIsModalOpen] = useState<boolean>(false)
+    const [wasModalOpen, setWasModalOpen] = useState(false)
```

Comment:
Since we don't use wasModalOpen to trigger a re-render but rather just to check a boolean value, maybe it might be better to convert it into a ref instead?

Reply:
This worked the best in this situation